schema/spacewalk/upgrade/susemanager-schema-4.1.10-to-susemanager-schema-4.1.11/001-fix_suseChannelUserRoleView.sql

```diff
@@ -0,0 +1,82 @@
+-- Copyright (c) 2020 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+
+DROP VIEW IF EXISTS suseChannelUserRoleView;
+
+CREATE OR REPLACE VIEW
+suseChannelUserRoleView
+AS
+  SELECT
+    combination.channel_id,
+    combination.user_id,
+    combination.role,
+    combination.user_org_id AS org_id,
+    combination.parent_channel_id,
+    CASE
+      -- if channel is in an org trusted by the user's org, and if the channel itself is shared between the two, then user can subscribe it
+      WHEN combination.role = 'subscribe' AND sharedChannels.is_shared_channel IS NOT NULL
+        THEN NULL
+      -- otherwise, if channel is not in user's org, then user can't manage it
+      WHEN combination.role = 'manage' AND combination.channel_org_id IS NULL OR combination.channel_org_id <> combination.user_org_id
+        THEN 'channel_not_owned'
+      -- otherwise, if channel is in a family without permissions for the user's org, then user can't subscribe it
+      WHEN combination.role = 'subscribe' AND channelPermissions.is_channel_available IS NULL
+        THEN 'channel_not_available'
+      -- otherwise, if channel is in a family without permissions for the user's org, then user can't subscribe it
+      WHEN adminUsers.is_admin_user IS NOT NULL
+        THEN NULL
+      -- otherwise, if channel does not have the "not_globally_subscribable" bit set, user can subscribe it
+      WHEN combination.role = 'subscribe' AND notGloballySubscribableChannels.is_not_globally_subscribable IS NULL
```

Comment:
I basically run: `sed -i -e 's/notGlobSubsChannels/notGloballySubscribableChannels/g' *` Rest of the file is the same except the comments above (copyright updated, mention to oracle equivalent removed)